### PR TITLE
resource/alicloud_db_instance: Fix set server_cert and client_ca_cert is sensitive

### PR DESCRIPTION
**Pull request by zwc01090043**
… is sensitive

### CODE REVIEW

File: alicloud/resource_alicloud_db_instance.go [hunk 1]
```diff
@@ -1030,6 +1032,7 @@ func resourceAliCloudDBInstanceUpdate(d *schema.ResourceData, meta interface{})
 				if v, ok := d.GetOk("client_ca_cert"); ok && v.(string) != "" {
 					request["ClientCACert"] = v.(string)
 				}
+				addDebug("ModifyDBInstanceSSLlll", "client_ca_cert", request["ClientCACert"])
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_db_instance.go:1035]: 去掉无用的日志

Comment (zwc01090043) [reply alicloud/resource_alicloud_db_instance.go]: 已删除

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Comment by zwc01090043** [comment]
image

**Pull request merged**